src/pages/editor/ChannelClaimFilter.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, FormControl, InputLabel, MenuItem, Select } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useSearchParams } from 'react-router';
+import { ChannelClaimParams } from '../../api/searchApi';
+import { dataTestId } from '../../utils/dataTestIds';
+
+const selectLabelId = 'select-label-id';
+
+export const ChannelClaimFilter = () => {
+  const { t } = useTranslation();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  return (
+    <Box sx={{ display: 'flex', gap: '1rem', my: '1rem', justifyContent: 'end' }}>
```

Comment:
trenger ikke gap om dette bare er for å flytte feltet til høyre
```suggestion
    <Box sx={{ display: 'flex', my: '1rem', justifyContent: 'end' }}>
```